<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EvisionCreateCubePolyClass</class>
 <widget class="QDialog" name="EvisionCreateCubePolyClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Cube Poly</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QLabel" name="label_6">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="text">
      <string>Ply文件存储位置:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="5">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="6" column="0">
       <widget class="QLabel" name="label_5">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>高(P1-P5):</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>请输入box的长宽高:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_3">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>长(P1-P2):</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_height"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>宽(P1-P3):</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_width"/>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="lineEdit_length"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QPushButton" name="pushButton_cancel">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
   <item row="3" column="5">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" rowspan="3" colspan="2">
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>345</width>
       <height>270</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="EvisionPolyTracker.qrc">:/EvisionPolyTracker/resource/Cube.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="6">
    <widget class="QLineEdit" name="lineEdit_savepath"/>
   </item>
   <item row="2" column="5">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="pushButton_confirm">
     <property name="text">
      <string>确定</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="EvisionPolyTracker.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_confirm</sender>
   <signal>clicked()</signal>
   <receiver>EvisionCreateCubePolyClass</receiver>
   <slot>onPush_confirm()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>422</y>
    </hint>
    <hint type="destinationlabel">
     <x>530</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>EvisionCreateCubePolyClass</receiver>
   <slot>onPush_cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>641</x>
     <y>417</y>
    </hint>
    <hint type="destinationlabel">
     <x>635</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onPush_confirm()</slot>
  <slot>onPush_cancel()</slot>
 </slots>
</ui>
